src/extension.ts

```diff
@@ -149,6 +150,10 @@ export async function activate(context: vscode.ExtensionContext) {
 		}
 	}
 
+	if (!_environment) {
+		throw new Error("_environment not set.");  // This should never happen, it is here just for TypeScript to be happy.
+	}
```

Comment:
what about to resolve this issue in same manner, as `!client` below?

-> return undefined;